WalletWasabi/WabiSabi/Backend/Rounds/RoundParameters.cs

```diff
@@ -120,10 +125,25 @@ public Transaction CreateTransaction()
 	/// <remarks>It won't be smaller than min allowed output amount.</remarks>
 	public Money CalculateMinReasonableOutputAmount()
 	{
-		var minEconomicalOutput = MiningFeeRate.GetFee(
-						Math.Max(
-							ScriptType.P2WPKH.EstimateInputVsize() + ScriptType.P2WPKH.EstimateOutputVsize(),
-							ScriptType.Taproot.EstimateInputVsize() + ScriptType.Taproot.EstimateOutputVsize()));
+		var minEconomicalOutput = MiningFeeRate.GetFee(MaxVsizeInputOutputPair);
 		return Math.Max(minEconomicalOutput, AllowedOutputAmounts.Min);
 	}
+
+	/// <returns>Smallest effective denom that's larger than min reasonable output amount. </returns>
+	public Money CalculateSmallestReasonableEffectiveDenomination()
+	{
+		var smallestEffectiveDenom = DenominationBuilder.CreateDenominations(
+				CalculateMinReasonableOutputAmount(),
+				AllowedInputAmounts.Max,
```

Comment:
This should be `AllowedOutputAmounts.Max`.

Reply:
What's wrong with me... Fixed.